src/NodeVisitor/StaticVariablesCollector.php

```diff
@@ -61,7 +61,13 @@ public function enterNode(Node $node)
                     $value = null;
                 }
 
-                $this->staticVariables[$staticVariable->name] = $value;
+                if ($staticVariable->var->name instanceof Node\Expr) {
```

Comment:
Just curious when name for static variable can be an expression?

Reply:
[StaticVar](https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Stmt/StaticVar.php) is now a wrapper for a [Variable](https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Expr/Variable.php). The last one in turn can be defined dynamically (`$a = 'foo'; $$a = 'bar';`). Frankly, I don't know any cases when such a dynamic nature can be applied to static vars so yeah, it's a kind of overengenering here :)